### PR TITLE
Zakharov week5

### PR DESCRIPTION
**Pull request by vzschool**

### CODE REVIEW

**Comment by panzerama** [comment]
It's not a huge deal, but because you're working in a personal repo you no longer need your name in the branch you submit, just the week designation.

File: client/src/App.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import Button from "./components/Button/Button"
+
+const cardLinks = ['Card A', 'Card B', 'Card C', 'Card D', 'Card E', 'Card F', 'Card G']
+
+function App() {
+  return (
+    <React.Fragment>
+      <Topbar />
+      <div className="page">
+        <div className="sidebar">
```
Comment (panzerama) [inline client/src/App.js]: What parts of this can be broken up into components?

File: client/src/components/Button/Button.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './Button.css';
+
+function Button(props) {
+    return (
+        // <button onClick={activateLasers}>
```
Comment (panzerama) [inline client/src/components/Button/Button.js]: don't forget to remove unnecessary comments before submitting a PR

File: client/src/App.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        </div>
+        <div className="content">
+          <Card />
+          <div className="navigation">
+            <Button text="Back"/>
+            <Button text="Flip"/>
+            <Button text="Next"/>
+          </div>
+        </div>
+        <div className="sidebar">
```
Comment (panzerama) [inline client/src/App.js]: what purpose does this sidebar serve?

File: client/src/components/Card/Card.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./Card.css";
+
+function Card() {
+  return (
+    <div className="card">
+      <div>
+        <img src="http://placekitten.com/g/200/300" className="img"></img>
```
Comment (panzerama) [inline client/src/components/Card/Card.js:8]: Can you think of why this image might be distorted in your site?

File: client/src/components/Topbar/Topbar.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import './Topbar.css';
+
+function Topbar() {
+    return <div className='topbar'>
```
Comment (panzerama) [inline client/src/components/Topbar/Topbar.js]: It is good practice to wrap the return content from a function in parentheses when returning multi-line jsx.

**Review by panzerama — CHANGES_REQUESTED** [review]

File: client/src/components/Card/Card.js (hunk 4 above)
Comment (vzschool) [reply client/src/components/Card/Card.js:8]: I think it is okay, I applied mods in css file

**Pull request merged**